test/MeshKernelNETTest/Api/MeshKernelTest.cs

```diff
@@ -1056,6 +1056,38 @@ public void Mesh2dDeleteEdgeThroughAPI()
                     api.DeallocateState(id);
                 }
             }
+        }
+
+        [Test]
+        public void Mesh2dDeleteEdgeByIndexThroughAPI()
+        {
+            //Setup
+
+            using (DisposableMesh2D mesh = CreateMesh2D(11, 11, 100, 100))
+            using (var api = new MeshKernelApi())
+            {
+                var id = 0;
+                var mesh2D = new DisposableMesh2D();
+                try
```

Comment:
this try-finally block could be replaced by a using(){} block, right? No exceptions are thrown here, are there?